rebar.config

```diff
@@ -43,7 +43,10 @@
 {profiles, [
     {test, [{deps, [meck,
                     {eunit_formatters, ".*", {git, "git://github.com/seancribbs/eunit_formatters", {tag, "v0.5.0"}}}]}]},
-    {eqc, [{deps, [meck]}]}
+    {eqc, [{deps, [meck]}]},
+    {gha, [{deps, [meck,
```

Comment:
I think you should not need the neck stuff. Unit will by default run the test profile and hence will use all in test as well as gha profile in above definition of "Run unit".

Reply:
Removed deps from the gha profile